setup.py

```diff
@@ -1,8 +1,15 @@
 #!/usr/bin/env python
 # -*- coding: utf-8 -*-
-from setuptools import setup
+from setuptools import find_packages, setup
+
 from djangocms_video import __version__
 
+
+REQUIREMENTS = [
+    'django-cms>=3.3.1',
```

Comment:
@FinalAngel 
If you want to support django 1.6 then you need django cms 3.0 as minimum.
I don't recommend it though, will add more overhead for us to keep maintaining all those versions.